### DISCUSSION TITLE
Bunk Sideloading APKs

### DISCUSSION THREAD

**Discussion by DarinStephen** [Nain57/Smart-AutoClicker#265] [General Feature Requests]
So I sideload alot of apps by either apk or apkx, and wanted to have the clicker be able to click the install button for the apk.

But It seems that when ever I try to record the condition to install an apk,  the overlay/recording will temporarily pause, or like the overlay will disappear.
But when I'm done installing or cancel the install, the overlay would come right back.

**Comment by Nain57** [comment]
Hi and thanks for your contribution.
This is not an issue, but a security from Android to avoid malicious installation of software.

There is nothing that can be done for that.
